route-rs-runtime/src/packet/udp.rs

```diff
@@ -0,0 +1,70 @@
+use crate::packet::*;
+
+pub struct UdpSegment<'packet> {
```

Comment:
Not sure "segment" is the semantically-correct name for this -- `UdpDatagram` maybe?